task/reindex.go

```diff
@@ -241,11 +302,33 @@ func docTransformer(ctx context.Context, tracker *Tracker, errChan chan error, e
 		}
 		wg.Wait()
 		close(transformedChan)
-		log.Info(ctx, "finished transforming docs")
+		log.Info(ctx, "finished transforming zebedee docs")
 	}()
 	return transformedChan
 }
 
+func resTransformer(ctx context.Context, tracker *Tracker, errChan chan error, resourceChan chan upstreamModels.Resource, maxTransforms int, topicsMapChan chan map[string]Topic) chan Document {
```

Comment:
```suggestion
func resourceTransformer(ctx context.Context, tracker *Tracker, errChan chan error, resourceChan chan upstreamModels.Resource, maxTransforms int, topicsMapChan chan map[string]Topic) chan Document {
```
Just to differentiate against 'responseTransformer'

Reply:
👍 done